common/src/main/java/tachyon/security/PlainSaslHelper.java

```diff
@@ -29,10 +40,50 @@
   }
 
   /**
-   * @param name the name of the provider
    * @return true if the provider was registered
    */
   public static boolean isPlainSaslProviderAdded() {
     return Security.getProvider(PlainSaslServerProvider.PROVIDER) != null;
   }
+
+  /**
+   * Get a ClientTransport for Plain
+   * @param username User Name of PlainClient
+   * @param password Password of PlainClient
+   * @param wrappedTransport The original Transport
+   * @return Wrapped Transport with Plain
+   * @throws SaslException
+   */
+  public static TTransport getPlainTransport(String username, String password,
```

Comment:
@apc999  Yes, it should be called *getPlainClientTransport*, thank you for your comments.